src/CoreWCF.BuildTools/tests/OperationParameterInjectionGeneratorTests.cs

```diff
@@ -1623,9 +1666,134 @@ public partial class IdentityService : IIdentityService
         }
 
         [Theory]
-        [InlineData("System.ServiceModel")]
-        [InlineData("CoreWCF")]
-        public async Task ShouldRaiseCompilationErrorWhenOperationContractIsAlreadyImplemented(string attributeNamespace)
+        [MemberData(nameof(GetTestVariations))]
+        public async Task ShouldNotRaiseCompilationErrorWhenServiceImplementationIsNotPartialButImplementedInterfaceIsNotAServiceContract(string attributeNamespace, string attribute)
+        {
+            var test = new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources =
+                    {
+@$"
+namespace MyProject
+{{
+    //[{attributeNamespace}.ServiceContract]
+    public interface IIdentityService
```

Comment:
Does leaving the attribute in the code but commented out have any purpose?

Reply:
Not really. It was there to make things more explicit. I just deleted.